.travis.yml

```diff
@@ -13,6 +13,7 @@ install:
   - go get github.com/zeromq/gomq
   - go get github.com/google/uuid
   - go get github.com/olekukonko/tablewriter
+  - go get github.com/stretchr/testify/assert
```

Comment:
I'm familiar with stretchr/testify for all of my testing needs; it's got almost anything you could want in a testing package